File: compiled_lang/cs/e10.branch.cs
using System;

class E10Branch
{
    static void Main()
    {
        Console.Write("Would you like a toast? [Yes/No]: ");
        string response = Console.ReadLine();

        // C#'s ternary operator
        string message = response == "Yes" ? "That's great!" : "How about a muffin?";

        Console.WriteLine(message);
    }
}

File: compiled_lang/rust/src/e10.branch.rs
use std::io::{self, Write};

fn main() {
    print!("Would you like a toast? [Yes/No]: ");
    io::stdout().flush().unwrap();

    let mut response = String::new();
    io::stdin().read_line(&mut response).unwrap();
    let response = response.trim_end();

    // Rust has no `?:` operator - if/else is itself an expression, which
    //  is how Rust normally replaces a ternary
    let message = if response == "Yes" { "That's great!" } else { "How about a muffin?" };

    println!("{}", message);
}
